### PR TITLE
18MS: Initial draft

### PR DESCRIPTION
**Pull request by perwestling**
In the initial draft each player (in reverse order) buy one company
each for face value. The remaining companies, if any, is discarded.

### CODE REVIEW

File: lib/engine/step/g_18_ms/draft_distribution.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
```
Comment (jenf) [inline lib/engine/step/g_18_ms/draft_distribution.rb:1]: Can any of the 1846 code be shared?

Comment (perwestling) [reply lib/engine/step/g_18_ms/draft_distribution.rb:1]: I do not know if that would be good? 
The 18MS draft is a straightforward one, where each player buy one private - they cannot pass. It is open. Variants of this exists in 1893 and 1876.
Possibly 1848 and 18CZ could reuse this type of draft.

So in the same way as we have several different auction maybe we need to have several different drafts?

File: assets/app/view/game_page.rb [hunk 2]
```diff
@@ -251,6 +251,8 @@ def render_action
         end
       when Engine::Round::Operating
         h(Game::Round::Operating, game: @game)
+      when Engine::Round::G18MS::Draft
```
Comment (tobymao) [inline assets/app/view/game_page.rb:254]: can we at least do Round::Draft share?

Comment (tobymao) [reply assets/app/view/game_page.rb:254]: you should be able to just when Draft

Comment (perwestling) [reply assets/app/view/game_page.rb]: Changed to Engine::Round::Draft

File: lib/engine/step/g_18_ms/draft_distribution.rb (hunk 1 above)
Comment (perwestling) [reply lib/engine/step/g_18_ms/draft_distribution.rb]: Draft from 1846 is now shared, and moved up one level.

**Comment by perwestling** [comment]
@tobymao Now it has been updated and rebased.

**Pull request merged**